extensions/editor.js

```diff
@@ -5,6 +5,7 @@ import './shared/public-path';
 import './shared/block-category';
 import './shared/plan-upgrade-notification';
 import './shared/stripe-connection-notification';
+import './shared/videopress-plan-check';
```

Comment:
Is it appropriate to load this check here? Another option might be to load these changes inside of the VideoPress block code, since this is already checking for a core video block and modifying it.

Reply:
This is resolved with an upcoming refactor.